charts/hivemq-platform-operator/values.schema.json

```
@@ -217,6 +217,21 @@
           }
         }
       }
+    },
+    "crd" : {
+      "type" : "object",
+      "properties" : {
+        "apply" : {
+          "type" : "boolean"
+        },
+        "waitReady" : {
+          "type" : "boolean"
+        },
+        "waitTimeout" : {
+          "type" : "string",
+          "pattern": "^P(?:(?:\\d+Y)?(?:\\d+M)?(?:\\d+D)?(?:T(?:\\d+H)?(?:\\d+M)?(?:\\d+(?:\\.\\d+)?S)?)?)$"
```

The following should work as this was introduced in the JSON schema draft 2019-09. However, current JSON schema validation for Helm is older (Draft 7).
```suggestion
          "format": "duration"
```
There is a pending [PR](https://github.com/helm/helm/pull/13283) raised a few days back aiming to update the JSON schema in Helm from Draft 7 to 2019-09 and 2020-12. Hopefully once it is merged, we could make use of it.

---

Nice finding! As discussed on Slack, we probably cannot use this right away though, because customers might still be on older Helm binaries. So this regex will probably live for quite a while.